src/sentry/dynamic_sampling/lastest_release_booster.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+from django.core.cache import cache
+from pytz import UTC
+
+from sentry.utils import json
+
+BOOSTED_RELEASE_TIMEOUT = 60 * 60
+ONE_DAY_TIMEOUT = 60 * 60 * 24
+
+
+def generate_dynamic_sampling_lock_key(project_id):
+    return f"dynamic-sampling-boost-lock:{project_id}"
+
+
+def generate_cache_key_for_observed_release(project_id, release_id):
+    """
+    Generates a cache key for releases that had a transaction observed in the last 24 hours
+    """
+    return f"ds::p:{project_id}:r:{release_id}:24h"
```

Comment:
The timeouts are currently set to 24h and 1h respectively, but are changed through a constant. The `:24h` also seems redundant at the moment, so could that be removed? 

Same below for `1h`.